classes/class-wc-connect-nux.php

```diff
@@ -83,5 +94,168 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			);
 			return $pointers;
 		}
+
+		public function get_jetpack_install_status() {
+			$status = Jetpack_Install_Status::UNINSTALLED;
+
+			if( 0 === validate_plugin( 'jetpack/jetpack.php' ) ) {
```

Comment:
Ugh, apologies. Didn't run the linter before pushing. Fixed all linting issues & rebased :)